### PR TITLE
Hanukkah embed 

### LINKED ISSUE #74: [Feature Request] Hanukkah Embed
While Hannukah is ongoing, when this command is invoked the bot should respond with an ASCII (or emoji) menorah depicting where we are in the holiday.

If the holiday hasn't yet begun, the bot should respond with how much time is left before it begins.

https://en.wikipedia.org/wiki/Hanukkah

### PR DESCRIPTION
**Pull request by RohanJnr**
closes #74 

Screenshot (173)
Screenshot (174)
Screenshot (175)

Pending tasks :
--> Use aiohttp insted of requests.

### CODE REVIEW

File: bot/seasons/christmas/hanukkah_embed [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import requests
+import datetime
+import logging
+
+
+log = logging.getLogger(__name__)
+
+HANUKKAH_DAYS = []
+HANUKKAH_MONTHS = []
+HANUKKAH_YEARS = []
```
Comment (jodth07) [inline bot/seasons/christmas/hanukkah_embed]: Check on the naming conventions. Capitalized for constants. 
Also, these variables can be initiated in `__init__` as well.

File: bot/seasons/christmas/hanukkah_embed [hunk 2]
```diff
@@ -0,0 +1,110 @@
+from discord.ext import commands
+from discord import Embed
+import requests
+import datetime
+import logging
+
```
Comment (jodth07) [inline bot/seasons/christmas/hanukkah_embed]: Update the imports order accordingly

File: bot/seasons/christmas/hanukkah_embed [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+class HanukkahEmbed(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.url = """https://www.hebcal.com/hebcal/?v=1&cfg=json&maj=on&min=on&mod=on&nx=on&year=now&month=x&ss=on&
+                      mf=on&c=on&geo=geoname&geonameid=3448439&m=50&s=on"""
+        self.hanukkah_dates = self.get_hanukkah_dates()
+
+    def get_hanukkah_dates(self):
```
Comment (jodth07) [inline bot/seasons/christmas/hanukkah_embed]: doc string for every methods and classes

File: bot/seasons/christmas/hanukkah_embed [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                                     f"Hanukkah will start at sundown on {hanukkah_start_day}th "
+                                     f"of {festival_starting_month}.")
+            else:
+                festival_end_month = hanukkah_end.strftime('%B')
+                embed.description = (f"Looks like you missed Hanukkah !"
+                                     f"Hanukkah ended on {hanukkah_end_day}th of {festival_end_month}.")
+
+            await ctx.send(embed=embed)
+
+    def hanukkah_dates_split(self):
```
Comment (jodth07) [inline bot/seasons/christmas/hanukkah_embed]: doc string here as well

File: bot/seasons/christmas/hanukkah_embed (hunk 1 above)
Comment (RohanJnr) [reply bot/seasons/christmas/hanukkah_embed]: okay

File: bot/seasons/christmas/hanukkah_embed (hunk 2 above)
Comment (RohanJnr) [reply bot/seasons/christmas/hanukkah_embed]: okay

File: bot/seasons/christmas/hanukkah_embed (hunk 3 above)
Comment (RohanJnr) [reply bot/seasons/christmas/hanukkah_embed]: on it

File: bot/seasons/christmas/hanukkah_embed.py [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+log = logging.getLogger(__name__)
+
+
+class HanukkahEmbed(commands.Cog):
+    """
+    A cog that returns information about Hanukkah festival.
+    """
+    def __init__(self, bot):
+        self.bot = bot
+        self.url = """https://www.hebcal.com/hebcal/?v=1&cfg=json&maj=on&min=on&mod=on&nx=on&year=now&month=x&ss=on&
```
Comment (MarkKoz) [inline bot/seasons/christmas/hanukkah_embed.py]: Use parenthesis here instead of triple quotes to be more consistent with the rest of the code.

Comment (RohanJnr) [reply bot/seasons/christmas/hanukkah_embed.py]: okay

File: bot/seasons/christmas/hanukkah_embed.py [hunk 6]
```diff
@@ -0,0 +1,113 @@
+import datetime
+import logging
+
+import requests
```
Comment (sco1) [inline bot/seasons/christmas/hanukkah_embed.py]: `requests` is blocking. You should use `aiohttp` instead, which is already installed as a dependency of d.py.

The [`aiohttp` quickstart](https://aiohttp.readthedocs.io/en/stable/client_quickstart.html) has an example of the equivalent request syntax.

File: bot/seasons/christmas/hanukkah_embed.py [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+        hanukkah_start = datetime.date(hanukkah_start_year, hanukkah_start_month, hanukkah_start_day)
+        hanukkah_end = datetime.date(hanukkah_end_year, hanukkah_end_month, hanukkah_end_day)
+        today = datetime.date.today()
+        # today = datetime.date(2019, 12, 24) (for testing)
+        day = str(today.day)
+        month = str(today.month)
+        year = str(today.year)
+        embed = Embed()
+        embed.title = 'Hanukkah Embed'
```
Comment (scragly) [inline bot/seasons/christmas/hanukkah_embed.py]: I'd suggest dropping `Embed` from the title.
```suggestion
        embed.title = 'Hanukkah'
```

**Comment by sco1** [comment]
Now that the constants have been updated (#198) this will need a minor merge conflict resolution before it can be merged.

**Pull request merged**